Libs/MRML/Core/vtkMRMLVolumeArchetypeStorageNode.cxx

```diff
@@ -281,6 +282,39 @@ void ApplyImageSeriesReaderWorkaround(vtkMRMLVolumeArchetypeStorageNode * storag
     }
   }
 }
+
+//----------------------------------------------------------------------------
+bool IsVolumeCoordinateSystemLeftHanded(const vtkSmartPointer<vtkMatrix4x4>& ijkToRASMatrix)
+{
+  // Check if the determinant of the direction matrix is less than 0.
+  vtkNew<vtkMatrix3x3> direction;
+  for(int i=0; i<3; i++)
```

Comment:
I think it would be good to keep spaces, like  `for(int i = 0; i < 3; i++)`

Reply:
Thx, I just reformatted the code.